bcda/cclf/cclf.go

```diff
@@ -56,23 +66,10 @@ type CclfImporter struct {
 	FileProcessor CclfFileProcessor
 }
 
-func (importer CclfImporter) importCCLF0(ctx context.Context, fileMetadata *cclfFileMetadata) (map[string]cclfFileValidator, error) {
-	if fileMetadata == nil {
```

Comment:
This nil check is now part of `LoadCclfFiles`. cclfZipMetadata is typed to ensure that both the CCLF0 and CCLF8 file exists before calling `importCCLF0` and `importCCLF8`.